redis/example/kustomization.yaml

```diff
@@ -2,14 +2,30 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-  - ssh://github.com/utilitywarehouse/shared-kustomize-bases//redis/manifests/dev-enablement/redis-shared?ref=v0.0.0
+  - ssh://github.com/utilitywarehouse/shared-kustomize-bases//redis/manifests?ref=v0.0.0
+  #- ../manifests
 
 
-# Resource requests and limits
+# Add prefix to name
+namePrefix: service-cache-
+
+# OpsLevel annotations
+commonAnnotations:
+ "app.uw.systems/description": "cache for some service"
+ "app.uw.systems/tier": "tier_4"
+
 patches:
+  # replace namespace
+  - target:
+      kind: .*
```

Comment:
[applied](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/14/commits/68d08076192f871ea58d454ee525adc6899e4284#diff-29fa0c5dc8aa4d8d05f166a67bcef458ae148967740033b27d20f1313a3e43dcR20)

Reply:
> [applied](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/14/commits/68d08076192f871ea58d454ee525adc6899e4284#diff-29fa0c5dc8aa4d8d05f166a67bcef458ae148967740033b27d20f1313a3e43dcR20)

Probably needs another update since https://github.com/utilitywarehouse/shared-kustomize-bases/pull/14/commits/9b70d78d5ed89533ec012047de0793285649ad2a to `namespace: change-me`